lib/govuk_publishing_components/presenters/subscription_links_helper.rb

```diff
@@ -14,13 +14,13 @@ def initialize(local_assigns)
       def email_signup_link_text
         return @local_assigns[:email_signup_link_text] if @local_assigns[:email_signup_link_text]
 
-        I18n.t("govuk_component.subscription_links.email_signup_link_text", default: "Get email alerts")
+        I18n.t("components.subscription_links.email_signup_link_text")
```

Comment:
Note to self: confirmed these locales are available in our apps due to [[1](https://github.com/alphagov/govuk_publishing_components/blob/9a5c519fb53c628987f371413758fa794b3b7d9c/lib/govuk_publishing_components.rb#L46)], so agree the default should be unnecessary.